apm-sniffer/apm-toolkit-activation/apm-toolkit-opentracing-activation/src/main/resources/skywalking-plugin.def

```diff
@@ -1,2 +1,2 @@
-org.skywalking.apm.toolkit.activation.opentracing.span.SkyWalkingSpanActivation
-org.skywalking.apm.toolkit.activation.opentracing.tracer.SkyWalkingTracerActivation
+opentracing-0.20.x=org.skywalking.apm.toolkit.activation.opentracing.span.SkyWalkingSpanActivation
+opentracing-0.20.x=org.skywalking.apm.toolkit.activation.opentracing.tracer.SkyWalkingTracerActivation
```

Comment:
`opentracing` activation is based on our application toolkit, so the name should not include version, especially ot version. Keep the name just `opentracing`